api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -71,8 +73,17 @@ String getImagesBucketName() {
   }
 
   private String readToString(String bucketName, String objectPath) {
+    return new String(getBlob(bucketName, objectPath).getContent()).trim();
+  }
+
+  // wrapper for storage.get() which throws NotFoundException instead of NullPointerException
```

Comment:
previously we used `storage.get().whatever()` without null checking.  it's still an error now, but a nicer one